doc/modules/feature_extraction.rst

```diff
@@ -937,7 +937,10 @@ Some tips and tricks:
         >>> print(CustomVectorizer().build_analyzer()(u"color colour")) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
         [...'color', ...'color']
 
-    The same approach could be used for stemming.
+    The same approach could be used for stemming or normalizing all numbers
```

Comment:
[...] for other styles of preprocessing; examples include stemming, lemmatization, or normalizing numerical tokens, with the latter illustrated in: